Sources/CleverBird/chat/streaming/StreamOptions.swift

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
Please remove this comment header. These comment headers that repeat the name of the file are just noise and don't add any value to the code.